app/notifications/notifications.component.css

```diff
@@ -0,0 +1,7 @@
+.filter-options {
+  color: #00205B;
+}
+
+.select-layout {
+  width: 250px;
```

Comment:
I set the width separately for notification type because when you select long text like 'Inhalt wurde Ihnen zugewiesen' , the wrap text gets broken.